ui/src/app/views/admin-user-bypass/component.tsx

```diff
@@ -21,114 +21,88 @@ const styles = reactStyles({
   }
 });
 
-export interface AccessModules {
-  complianceTraining: boolean;
-  betaAccess: boolean;
-  eraCommons: boolean;
-  twoFactorAuth: boolean;
-  dataUseAgreement: boolean;
-}
-
 export class AdminUserBypass extends React.Component<
     { profile: Profile},
-    { modules: AccessModules, editedModules: AccessModules,
+    { initialModules: AccessModule[], selectedModules: AccessModule[],
       open: boolean} > {
 
   constructor(props) {
     super(props);
+    const {profile} = props;
+    const initialModules = [
+      !!profile.betaAccessBypassTime ? AccessModule.BETAACCESS : null,
```

Comment:
I'm not crazy about sticking nulls into the array. I'd either filter them or restructure this code a bit, e.g. a bunch of `if `s or `&&` (or I'm sure there's something in lodash to make this easy)

Reply:
Yeah... I didn't really like this either. I will see if lodash has something better, but worst case I can always filter out the nulls after. 

I would prefer not to do them as ifs, but if a whole bunch of if statements are what is necessary, I can update.